src/wfgoto.cpp

```diff
@@ -482,17 +462,17 @@ VOID UpdateGotoList(HWND hDlg)
 
 	DWORD dw = GetDlgItemText(hDlg, IDD_GOTODIR, szText, COUNTOF(szText));
 
-	vector<PDNODE> *options = GetDirectoryOptionsFromText(szText, &bLimited);
+	vector<PDNODE> options = GetDirectoryOptionsFromText(szText, &bLimited);
 
 	HWND hwndLB = GetDlgItem(hDlg, IDD_GOTOLIST);
 	SendMessage(hwndLB, LB_RESETCONTENT, 0, 0);
 
-	if (options == NULL)
+	if (options.size() == 0)
 		return;
-		
-	for (int i = 0; i < 10 && i < options->size(); i++)
+
+	for (auto i = 0u; i < 10u && i < options.size(); i++)
 	{
-		GetTreePath(options->at(i), szText);
+		GetTreePath(options[i], szText);

```

Comment:
`operator[](int)` is not the same as `.at(int)` is this intended? This change removes bounds checking.